evennia/__init__.py

```diff
@@ -363,6 +364,9 @@ class SystemCmds(_EvContainer):
     del SystemCmds
     del _EvContainer
 
+    # Trigger EvRich to monkey-patch Rich in-memory.
+    install_evrich()
```

Comment:
This does not belong with the PR?

Reply:
Oops I don't know how that snuck in there.